src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -504,6 +504,11 @@ const MeasureGroups = () => {
                         renderInput={(params) => (
                           <TextField {...params} label="" />
                         )}
+                        onKeyDown={(e) => {
```

Comment:
this can be handled in a different way. Looks like whenever a new field is introduced, we need this code. I looked into it lil bit and seems like event is getting propaugated to `+ Add Measure Group` link(actually a button which I think should be a link). Having `type="button"` attribute on `add measure group` button would fix this or even you can replace button element with with `DSLink` from the design system.

Reply:
@adongare @nmorasb Thanks so much Nate and Ashok! I will change the button to DSLink and revert some some changes for the fix before.